options.js

```diff
@@ -25,34 +49,63 @@ submitButton.onclick = function() {
     return;
   }
 
-  setExtensionColor(color);
-  log("Color changed to " + color);
+  if(color === crunchyrollOrange) {
+    confirmationMessage.innerText = "You can't remove this color!";
+    log("Tried to remove theme color");
+    return;
+  }
+
+  getColorMenu().then( colorOptions => {
+    const isInMenu = colorOptions.includes(color);
+
+    if(!isInMenu) {
+      confirmationMessage.innerText = "This color isn't in the menu";
+      log("Color not in menu");
+      return;
+    }
+
+    colorOptions = colorOptions.filter(function(element) { return element != color; });
+
+    getExtensionColor().then( currentColor => {
+      const isInMenu = colorOptions.includes(currentColor);
+      if(!isInMenu) setExtensionColor(crunchyrollOrange);
+    });
+
+    setColorMenu(colorOptions);
+    document.location.reload();
```

Comment:
Is it a page reload call? Isn't it better to create an update function instead?

Reply:
Created, it's the updateColorMenu() function